stopcovid/event_distributor.py

```diff
@@ -0,0 +1,26 @@
+from stopcovid.utils import dynamodb as dynamodb_utils
+from stopcovid.clients import sqs
+
+
+def _get_outbound_sms_messages(dialog_events):
+    # TODO: respond with the correct content
+    return [
+        sqs.OutboundSMSSchema().load(
```

Comment:
out of curiosity, why not construct the `OutboundSMS` directly instead of going through the schema

Reply:
I was trying to keep it consistent with the pattern we've used for instantiating objects elsewhere. Come to think of it, this should probably just be a dataclass instead. Updating now

Reply:
forgot about dataclasses!